main/src/kotlinx/team/infra/Utils.kt

```diff
@@ -42,10 +41,10 @@ class KotlinClosure1<T : Any?, V : Any>(
     fun doCall(it: T): V? = it.function()
 }
 
-fun <T> Any.closureOf(action: T.() -> Unit) : Closure<T> =
+internal fun <T> Any.closureOf(action: T.() -> Unit) : Closure<T> =
```

Comment:
I can't find any usages of `closureOf` and `KotlinClosure1` within the infra project. Is it safe to remove them?